packages/types/src/checkin.ts

```diff
@@ -38,14 +38,16 @@ export interface SerializedCheckIn {
   };
 }
 
-export interface CheckIn {
+export type CheckIn = {
   // The distinct slug of the monitor.
   monitorSlug: SerializedCheckIn['monitor_slug'];
+  // Check-In ID (unique and client generated).
+  checkInId?: SerializedCheckIn['check_in_id'];
```

Comment:
You could theoretically call captureCheckIn with status "in_progress" but not provide a checkInId. We should couple those types. Not coupling would be very opaque for users why things are not working.

Reply:
yup refined with 33fd221d1598e5dcc2946c24c90c21de1530094d

Reply:
I think that commit is on the wrong branch

Reply:
🤦